doc/cypher.py

```diff
@@ -34,20 +34,23 @@ def doit(server, api_token, query, format):
             j = r.json()
         except ValueError:
             sys.stderr.write('JSON syntax error\n')
-            print >>sys.stderr, r.text[0:1000]
+            print(r.text[0:1000], file=sys.stderr)
```

Comment:
why `file=` here? It *seems* to me this would set a variable called `file` that is then never used, but perhaps this is some syntax I'm not aware of (?).  ...Or perhaps this is intended to be `file: sys.stderr` ?